bambi/plots/effects.py

```diff
@@ -0,0 +1,402 @@
+        works when pps is False as the target may not be available in the posterior predictive
+        distribution.
+    pps: bool, optional
+        Whether to plot the posterior predictive samples. Defaults to ``False``.
+    use_hdi : bool, optional
+        Whether to compute the highest density interval (defaults to True) or the quantiles.
+    prob : float, optional
+        The probability for the credibility intervals. Must be between 0 and 1. Defaults to 0.94.
+        Changing the global variable ``az.rcParam["stats.hdi_prob"]`` affects this default.
+    transforms : dict, optional
+        Transformations that are applied to each of the variables being plotted. The keys are the
+        name of the variables, and the values are functions to be applied. Defaults to ``None``.
+
+    Returns
+    -------
+    cap_data : pandas.DataFrame
+        A DataFrame with the ``create_cap_data`` and model predictions.
+
+    Raises
+    ------
+    ValueError
+        If passed ``covariates`` is not in correct key, value format.
+        If length of ``covariates`` is not between 1 and 3.
+    """
+
+    covariate_kinds = ("main", "group", "panel")
+    if not isinstance(covariates, dict):
+        covariates = listify(covariates)
+        covariates = dict(zip(covariate_kinds, covariates))
+    else:
+        assert covariate_kinds[0] in covariates
+        assert set(covariates).issubset(set(covariate_kinds))
+
+    assert 1 <= len(covariates) <= 3
+
+    if transforms is None:
+        transforms = {}
+
+    if prob is None:
+        prob = az.rcParams["stats.hdi_prob"]
+
+    if not 0 < prob < 1:
+        raise ValueError(f"'prob' must be greater than 0 and smaller than 1. It is {prob}.")
+
+    cap_data = create_cap_data(model, covariates)
+    response_name = get_aliased_name(model.response_component.response_term)
+    response_transform = transforms.get(response_name, identity)
+
+    if pps:
+        idata = model.predict(idata, data=cap_data, inplace=False, kind="pps")
+        y_hat = response_transform(idata.posterior_predictive[response_name])
+        y_hat_mean = y_hat.mean(("chain", "draw"))
+    else:
+        idata = model.predict(idata, data=cap_data, inplace=False)
+        y_hat = response_transform(idata.posterior[f"{response_name}_{target}"])
+        y_hat_mean = y_hat.mean(("chain", "draw"))
+
+    if use_hdi and pps:
+        y_hat_bounds = az.hdi(y_hat, prob)[response_name].T
+    elif use_hdi:
+        y_hat_bounds = az.hdi(y_hat, prob)[f"{response_name}_{target}"].T
+    else:
+        lower_bound = round((1 - prob) / 2, 4)
+        upper_bound = 1 - lower_bound
+        y_hat_bounds = y_hat.quantile(q=(lower_bound, upper_bound), dim=("chain", "draw"))
+
+    lower_bound = round((1 - prob) / 2, 4)
+    upper_bound = 1 - lower_bound
+
+    cap_data["estimate"] = y_hat_mean
+    cap_data[f"lower_{lower_bound}%"] = y_hat_bounds[0]
+    cap_data[f"upper_{upper_bound}%"] = y_hat_bounds[1]
```

Comment:
I think these will look like `upper_0.97%` instead of `upper_97%`, right? Just need to multiply by 100 above.